breadcord/core_modules/modulemanager/__init__.py

```diff
@@ -75,6 +75,14 @@ async def install(self, interaction: discord.Interaction, module: str):
             return
 
         async with self.session.get(f'https://api.github.com/repos/{module}/contents/manifest.toml') as response:
+            if response.status != 200:
+                await interaction.response.send_message(embed=discord.Embed(
+                    colour=discord.Colour.red(),
+                    title='Invalid module!',
+                    description='The repository specified does not contain a `manifest.toml` file.'
```

Comment:
While this is certainly an improvement over what we've got before, it doesn't cover cases such as e.g. the GitHub repository not existing or GitHub being down. It'd probably be more helpful to change the error message to something like:
```
Module not found!

The repository specified does not exist, or isn't a Breadcord module.
```

In a future commit I could then improve upon that by making additional checks be performed if the `manifest.toml` file fails to be read.